src/xoak/accessor.py

```diff
@@ -224,16 +224,13 @@ def _get_pos_indexers(self, indices, indexers):
 
         return pos_indexers
 
-    def sel(
+    def query(
         self, indexers: Mapping[Hashable, Any] = None, **indexers_kwargs: Any
     ) -> Union[xr.Dataset, xr.DataArray]:
```

Comment:
I think the return type is rather `Dict[Hashable, Variable]`.

It would be nice if `query` would eventually return the distances too. Maybe returning `Tuple[DataArray, DataArray]` would make sense instead of trying to return both indexers and distances in the same xarray object. This is closer to what the underlying indexes usually return. Also `DataArray` (even without any coordinate) is more common than `Variable`. We can save this for later, though.